.changeset/olive-rats-rest.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/create-app': minor
+---
+
+Accessibility updates:
+
+- Added `aria-label` to the sidebar Logo link
```

Comment:
These changes are a bit tricky since they affect the app template. We usually describe how to apply these changes to existing apps as well, typically with a small diff. You can find examples of that in the [`create-app` changelog](https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md)